samples/tab-personal-sso-quickstart/js/src/components/App.js

```diff
@@ -19,17 +19,27 @@ import ClosePopup from "./ClosePopup";
 function App() {
 
   // Initialize the Microsoft Teams SDK
-  microsoftTeams.initialize();
+  microsoftTeams.app
+  .initialize()
```

Comment:
It can be in same line?

Reply:
we are getting warning on this also display one line on this code for the depreciation that's why we have added app.